compiler/src/main/java/com/xfinity/blueprint_compiler/BlueprintProcessor.java

```diff
@@ -109,7 +112,56 @@ public boolean process(Set<? extends TypeElement> annotations, RoundEnvironment
 
             String viewHolderClassName = ((Symbol.ClassSymbol) annotatedClass).fullname.toString();
 
-            componentViewInfoList.add(new ComponentViewInfo(viewType, viewHolderClassName));
+            Map<String, String> children = new HashMap<>();
+            List<String> methodNames = new ArrayList<>();
+            for (Element enclosedElement : annotatedElement.getEnclosedElements()) {
+                // We only look at fields
+                if (enclosedElement instanceof Symbol.MethodSymbol) {
+                    methodNames.add(enclosedElement.getSimpleName().toString().toLowerCase());
+                }
+            }
+
+            for (Element enclosedElement : annotatedElement.getEnclosedElements()) {
+                // We only look at fields
+                if (enclosedElement.getKind().isField()) {
+                    VariableElement variable = (VariableElement) enclosedElement;
+                    String variableName = enclosedElement.getSimpleName().toString();
+
+                    //TODO: in kotlin, none of the fields will be public, since vals are backed by
+                    // private final fields
+                    // Only do things with Public items
+                    // (because we set/get outside the ViewHolder in the view class)
+                    if (variable.getModifiers().contains(Modifier.PUBLIC)) {
```

Comment:
This if is checking for the Public modifier. If in kotlin that's default private, then we'll never check for the getter.  I think this if needs to be reversed.  If it's not public, look for the getter so we can proceed.

Reply:
Fixed.  Please take another look